services/wireguard/service/options.go

```diff
@@ -50,8 +50,12 @@ func ParseFlags(ctx *cli.Context) service.Options {
 }
 
 // ParseJSONOptions function fills in Openvpn options from JSON request
-func ParseJSONOptions(request json.RawMessage) (service.Options, error) {
+func ParseJSONOptions(request *json.RawMessage) (service.Options, error) {
+	if request == nil {
+		return Options{}, nil
```

Comment:
could use named return params here to avoid declaring the opts, and using naked returns instead.